app/Website/database/migrations/2024_03_16_174322_create_tags_table.php

```diff
@@ -13,6 +13,7 @@ public function up(): void
         Schema::create('blog_post_tags', function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->softDeletes();
```

Comment:
This would have to be added in a separate migration, this migration was already executed.